content/en/docs/Writing policies/jmespath.md

```diff
@@ -408,6 +408,100 @@ spec:
+
+`sum()` is value-aware (based on the formatting used for the inputs) just as `add()` and is capable of adding list of numbers, quantities, and durations without any form of unit conversion.
+
+Arithmetic filters like `sum()` currently accept inputs in the following formats.
+
+* Number (ex., \`10\`)
+* Quantity (ex., '10Mi')
+* Duration (ex., '10h')
+
+Note that how the inputs are enclosed determines how Kyverno interprets their type. Numbers enclosed in back ticks are scalar values while quantities and durations are enclosed in single quotes thus treating them as strings. Using the correct enclosing character is important because, in Kubernetes "regular" numbers are treated implicitly as units of measure. The number written \`10\` is interpreted as an integer or "the number ten" whereas '10' is interpreted as a string or "ten bytes". See the [Formatting](#formatting) section above for more details.
+
+| Input                 | Output   |
+|-----------------------|----------|
+| [Number]              | Number   |
+| [Quantity or Number]  | Quantity |
+| [Duration or Number]  | Duration |
+
+Some specific behaviors to note:
+
+* If a combination of duration ('1h') and  number (\`5\`) are the inputs, the number will be interpreted as seconds resulting in a sum of `1h0m5s`.
+* Because of durations being a string just like [resource quantities](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/), and the minutes unit of "m" also present in quantities interpreted as the "milli" prefix, there is no support for minutes.
+
+**Example:** This policy denies a Pod if the aggregated storage quota is greater than the one defined in configmap.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: restrict-storage-quota
+  annotations:
+    policies.kyverno.io/title: Restrict storage quota for volumes
+    policies.kyverno.io/severity: high
+    policies.kyverno.io/subject: Volume
+    policies.kyverno.io/minversion: 1.8.0
+    policies.kyverno.io/description: >-
+      Kubernetes namespaces can only accommodate a maximum capacity of
+      volume storage due to resources limitation for namespaces/tenants.
+      This policy restricts the capacity in Gigabytes of storage volumes as
+      per ConfigMap, so that the number of maximum allowed resources can be
+      modified on demand.
```

Comment:
Remove annotations, please. These just further bloat the page.

Reply:
applied the changes